src/rng/Counter_RNG.hh

```diff
@@ -11,6 +11,7 @@
 #ifndef Counter_RNG_hh
 #define Counter_RNG_hh
 
+#include "device/config.h"
 #include "rng/config.h"
```

Comment:
Does `rng/config.h` depend on the content of `device/config.h`? If so, I recommend moving the include from this location and into `rng/config.h`.

Reply:
Nope. I don't use anything from `device/config.h` in `rng/config.h`